api/to-do-db-source.json

```
@@ -59,4 +59,4 @@
       "id": 4
     }
   ]
-}
\ No newline at end of file
+}
```

```suggestion
}

```
Looks like you cut off an end-of-line character when you fixed the file.
Can you add it back and then squash your commits into one, please?
Thanks!